lib/index.js

```diff
@@ -1,8 +1,8 @@
 let buildBundler = require("./build-bundler");
-let buildConfig = require("./build-config");
+let path = require("path");
 
-module.exports = (rawConfig, configDir, { watcher, manager, compact, browsers }) => {
-	let config = buildConfig(rawConfig, configDir, compact, manager, browsers);
+module.exports = (pluginConfig, configDir, { watcher, manager, compact, browsers }) => {
+	let config = buildConfig(pluginConfig, configDir, { compact, manager, browsers });
```

Comment:
FWIW, minute detail: It'd be nice if the order of options here corresponded to that withing faucet-core (`{ manager, watcher, browsers, compact }` as of now).

Reply:
I'll adjust when you adjusted the signature